src/encoded/static/components/item-pages/components/AttributionTabView.js

```diff
@@ -5,9 +5,13 @@ import _ from 'underscore';
 import PropTypes from 'prop-types';
 import ReactTooltip from 'react-tooltip';
 import { object } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
-import { ItemFooterRow } from '@hms-dbmi-bgm/shared-portal-components/es/components/ui/ItemFooterRow';
+import { ItemFooterRow, ExternalReferenceLink } from '@hms-dbmi-bgm/shared-portal-components/es/components/ui/ItemFooterRow';
+import { StackedBlockTable, StackedBlock, StackedBlockList, StackedBlockName, StackedBlockNameLabel } from '@hms-dbmi-bgm/shared-portal-components/es/components/browse/components/StackedBlockTable';
 import { FormattedInfoBlock, WrappedCollapsibleList } from './FormattedInfoBlock';
 import { Publications } from './Publications';
+import { FileEntryBlock } from '../../browse/components/FileEntryBlock';
+import { expFxn, Schemas } from './../../util';
+import { experimentsFromExperimentSet } from '../../util/experiments-transforms';
```

Comment:
Codacy found an issue: ['experimentsFromExperimentSet' is defined but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5995039)